flang/lib/Lower/OpenMP.cpp

```diff
@@ -47,6 +47,18 @@ static void genObjectList(const Fortran::parser::OmpObjectList &objectList,
   }
 }
 
+template <typename Op>
+static void finalizeRegionOp(Op &op, Fortran::lower::FirOpBuilder &firOpBuilder,
```

Comment:
Nit: Would createRegionOp/createBodyOfOp be a better name?

Reply:
createBodyOfOp sounds better, since the actuall operation creation is outside this function. This function just ensures region related stuff.